projects/resources/MIYOO/mapping.xml

```diff
@@ -0,0 +1,53 @@
+<MAPPINGS>
+<!--
+DEFAULT MIYOO MAPPING
+shoulder, face, start/select buttons are doubled
+enable DUMPEVENT in config.xml and check log to help with mapping
+eg:
+<MAP src="input reported by OS" dst="virtual button" />
+-->
+    <!-- miyoo dpad -->
+    <MAP src="key:0:up" dst="/event/up" />
+    <MAP src="key:0:down" dst="/event/down" />
+    <MAP src="key:0:left" dst="/event/left" />
+    <MAP src="key:0:right" dst="/event/right" />
+    <!-- miyoo a -->
+    <MAP src="key:0:space" dst="/event/a" />
+    <!-- miyoo b -->
+    <MAP src="key:0:left ctrl" dst="/event/b" />
+    <!-- miyoo x -->
+    <MAP src="key:0:left shift" dst="/event/a" />
+    <!-- miyoo y -->
+    <MAP src="key:0:left alt" dst="/event/b" />
+    <!-- miyoo start -->
+    <MAP src="key:0:return" dst="/event/start" />
+    <!-- miyoo select -->
+    <MAP src="key:0:right ctrl" dst="/event/start" />
+    <!-- miyoo l1 -->
+    <MAP src="key:0:e" dst="/event/lshoulder" />
+    <!-- miyoo r1 -->
+    <MAP src="key:0:t" dst="/event/rshoulder" />
+    <!-- miyoo l2 -->
+    <MAP src="key:0:tab" dst="/event/lshoulder" />
+    <!-- miyoo r2 -->
+    <MAP src="key:0:backspace" dst="/event/rshoulder" />
+<!--
```

Comment:
Macro examples contain errors
`but` should be `key`
Button presses don't represent the actual button names